src/sentry/rules/actions/notify_event_service.py

```diff
@@ -142,10 +143,15 @@ def __init__(self, *args, **kwargs):
         self.form_fields = {
             "service": {
                 "type": "choice",
-                "choices": [[i.slug, i.title] for i in self.get_services()],
+                "choices": [[i.slug, self.transform_title(i.title)] for i in self.get_services()],
             }
         }
 
+    def transform_title(self, title):
+        if title not in PLUGINS_WITH_FIRST_PARTY_EQUIVALENTS:
+            return title
+        return "(Legacy) " + title
```

Comment:
I might actually write. the if statment the other way around to be more of a guard

```suggestion
        if title in PLUGINS_WITH_FIRST_PARTY_EQUIVALENTS:
            return f"(Legacy) {title}"
        return title
```